vwifi.c

```diff
@@ -33,7 +33,7 @@ struct owl_context {
 
     struct mutex lock;
     struct work_struct ws_connect, ws_disconnect;
-    char connecting_ssid[SSID_MAX_LENGTH];
+    char connecting_ssid[SSID_MAX_LENGTH + 1];
```

Comment:
Add the comment which indicates the space for null terminator.

Reply:
Ok. Added.